refinery/ui/source/js/commons/data-sets/search-api.spec.js

```diff
@@ -68,7 +68,7 @@ describe('DataSet.search-api: unit tests', function () {
       'hl.simple.post': '%3C%2Fem%3E',
       'hl.simple.pre': '%3Cem%3E',
       q: _query,
-      qf: 'title%5E0.5+accession+submitter+text',
+      qf: 'title%5E0.5+accession+submitter+text+description',
```

Comment:
You now search twice on the description because the description is part of the `text` field. The `text` field is composed of multiple other fields and make up the core part of the document to be search for in Solr. Check out `templates/searches/indexes/core/dataset_text.txt`